src/sentry/static/sentry/app/types/iOSDeviceList.tsx

```diff
@@ -8,15 +8,15 @@ type Options = {
 type Device = {
   Type: string;
   Generation: string;
-  ANumber: string[];
```

Comment:
Are these changes supposed to be in this PR?

Reply:
yeah I just thought I'd add them in here instead of doin a separate pr - can always pull them out if thats better

Reply:
Loks like a totally separate change right?

Reply:
It's not related to the component in this PR and is not covered by the description, pulling it out separately is better

Reply:
cool - pulled it out and put it in https://github.com/getsentry/sentry/pull/14267

Reply:
Thanks!